nikola/plugins/template/mako.py

```diff
@@ -65,7 +65,12 @@ def get_string_deps(self, text, filename=None):
         # Some templates will include "foo.tmpl" and we need paths, so normalize them
         # using the template lookup
         for i, d in enumerate(deps):
-            deps[i] = self.get_template_path(d)
+            dep = self.get_template_path(d)
+            if dep:
+                deps[i] = dep
+            else:
+                LOGGER.error("Cannot find template {0} referenced in {1}",
+                    d, filename)
```

Comment:
flake8 complains here: `nikola/plugins/template/mako.py:73:21: E128 continuation line under-indented for visual indent`